### PR TITLE
Added project logo

### PR DESCRIPTION
**Pull request by AndChurchville**

### CODE REVIEW

**Comment by AndChurchville** [comment]
Sure making the changes right now and I can make them SVG as well.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,14 @@ to encrypt the text github with an offset of 4, using a `Ceasar` cypher:
 
 You can test this application running the `pytest` command
 
+
+## Encryptor Logo
```
Comment (rafaeelaudibert) [inline README.md]: You can just add the main one in the header of the README (after the title). After that, I think we are good (and in awesome shape) to go!

Thank you for your time, it is a really great logo, appreciated it!

Comment (AndChurchville) [reply README.md]: Got it! Glad to help out!

**Comment by rafaeelaudibert** [comment]
Thanks dude, awesome job!

**Pull request merged**